guide/ops/high-availability/high-availability-supplemental.md

```diff
@@ -0,0 +1,142 @@
+
+{% highlight bash %}
+$ curl -u myusername:mypassword http://<ip-address>:8081/v1/server/ha/state
+{% endhighlight %}
+
+This will return one of the following states:
+
+{% highlight bash %}
+
+"INITIALIZING"
+"STANDBY"
+"HOT_STANDBY"
+"HOT_BACKUP"
+"MASTER"
+"FAILED"
+"TERMINATED"
+
+{% endhighlight %}
+
+Note: The quotation characters will be included in the reply
+
+To obtain information about all of the nodes in the cluster, run the following command against any of the nodes in the cluster:
+
+{% highlight bash %}
+$ curl -u myusername:mypassword http://<ip-address>:8081/v1/server/ha/states
+{% endhighlight %}
+
+This will return a JSON document describing the Brooklyn nodes in the cluster. An example of two HA Brooklyn nodes is as follows (whitespace formatting has been
+added for clarity):
+
+{% highlight yaml %}
+
+{
+  ownId: "XkJeXUXE",
+  masterId: "yAVz0fzo",
+  nodes: {
+    yAVz0fzo: {
+      nodeId: "yAVz0fzo",
+      nodeUri: "http://<server1-ip-address>:8081/",
+      status: "MASTER",
+      localTimestamp: 1466414301065,
+      remoteTimestamp: 1466414301000
+    },
+    XkJeXUXE: {
+      nodeId: "XkJeXUXE",
+      nodeUri: "http://<server2-ip-address>:8081/",
+      status: "STANDBY",
+      localTimestamp: 1466414301066,
+      remoteTimestamp: 1466414301000
+    }
+  },
+  links: { }
+}
+
+{% endhighlight %}
+
+The examples above show how to use `curl` to manually check the status of Brooklyn via its REST API. The same REST API calls can also be used by
+automated third party monitoring tools such as Monit 
+
+### Failover
+When running as a HA standby node, each standby Brooklyn server (in this case there is only one standby) will check the shared persisted state
+every 1 second to determine the state of the HA master. If no heartbeat has been recorded for thirty seconds, then an election will be performed
+and one of the standby nodes will be promoted to master. At this point all requests should be directed to the new master node
+
+In the event that tasks - such as the provisioning of a new entity - are running when a failover occurs, the new master will display the current
+state of the entity, but will not resume its provisioning or re-run any partially completed tasks. In this case it will usually be necesarry
+to remove the node and reprovision it
+
+### Client Configuration
+It is the responsibility of the client to connect to the master Brooklyn server. This can be accomplished in a variety of ways:
+
+* **Reverse Proxy**
+
+  To allow the client application to automatically fail over in the event of a master server becoming unavailable, or the promotion of a new master,
+  a reverse proxy can be configured to route traffic depending on the response returned by `http://<ip-address>:8081/v1/server/ha/state` (see above).
+  If a server returns `"MASTER"`, then traffic should be routed to that server, otherwise it should not be. The client software should be configured
```

Comment:
Add that it could take 30 seconds for the standby to be promoted, so the reverse proxy should retry for at least this period. Or that the failover time should be reconfigured to be shorter.